clang/include/clang/3C/RewriteUtils.h

```diff
@@ -254,6 +254,9 @@ class RewriteConsumer : public ASTConsumer {
   void emitRootCauseDiagnostics(ASTContext &Context);
 };
 
-bool canRewrite(Rewriter &R, SourceRange &SR);
+bool canRewrite(Rewriter &R, const SourceRange &SR);
+
+void rewriteSourceRange(Rewriter &R, const SourceRange &Range,
+                        const std::string &NewText);
```

Comment:
Since you weren't able to use this everywhere `ReplaceText` was, a brief description with usage would be helpful.

Reply:
Agree. (Is Kyle right that not all places were changed?)

Reply:
He was right when he commented, but I fixed that by overloading `rewriteSourceRange` accept a `CharSourceRange`. Now `ReplaceText` isn't used anywhere.